api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -96,10 +99,28 @@ message DownstreamTlsContext {
 }
 
 // TLS context shared by both client and server TLS contexts.
-// [#next-free-field: 9]
+// [#next-free-field: 11]
 message CommonTlsContext {
   option (udpa.annotations.versioning).previous_message_type = "envoy.api.v2.auth.CommonTlsContext";
 
+  // Config for CA client plugin to get secrets
+  message CAClientPluginConfig {
+    // plugin name
+    string name = 1;
+
+    // opaque name to identify a secret when workload needs more than one secret
+    string secret_name = 2;
+
+    // plugin specific config based on the plugin name
+    google.protobuf.Any typed_config = 3;
```

Comment:
When https://github.com/envoyproxy/envoy/pull/10908/files#r420136445 lands, we should switch to this here. This should happen before this PR merges.

Reply:
Okay, will use `TypedExtensionConfig` instead of `Any` once that one is in.